sktime/forecasting/base/_base.py

```diff
@@ -184,3 +186,18 @@ def get_fitted_params(self):
         fitted_params : dict
         """
         raise NotImplementedError("abstract method")
+
+    def _get_model_params(self):
```

Comment:
Why do we need this? Isn't this already handled through sklearn's hyper-parameter interface via `get_params`?

Reply:
I wrote it to get the params that have been passed when initializing an `_SktimeForecaster` child. This params are used by `Prophet` in the `update()` function to initialize the updated (actually a new) model. You are right, I can just use `get_params()`, so I will fix this.